test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +44,34 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+
+        // test single null value
+        assertAnyNull((Object)null);
+        assertAnyNull(null, 3, "Hello");
+
+        // test multiple null values
+        assertAnyNull(null, null, null);
+        assertAnyNull(3, null, true, null, "Hello", null);
+
+        // test single non null value
+        assertAnyNotNull("I am not null!");
+        assertAnyNotNull(5);
+        assertAnyNotNull(true);
+
+        // test multiple non null values
+        assertAnyNotNull(false, true);
+        assertAnyNotNull(3, 4, false, 0.123123, "Non null");
+
+    }
+
+    private void assertAnyNull(Object... objects) {
```

Comment:
It's good that you followed the style used by the code nearby.
However, in this case, there is no need to abstract the assertTrue(Utils.isAnyNull(...)) as a separate method. Asserting the result of the method itself is fine.
The existing test uses assertAreUnique() in order to prevent Arrays.asList(..) from being used each time.